images/cassandra/rootfs/run.sh

```diff
@@ -21,6 +21,8 @@ CASSANDRA_CFG=$CASSANDRA_CONF_DIR/cassandra.yaml
 
 mkdir -p $CASSANDRA_CONF_DIR_RO
 cp -L $CASSANDRA_CONF_DIR_RO/* $CASSANDRA_CONF_DIR
+chown -R 1000:1000 $CASSANDRA_CONF_DIR
```

Comment:
Why are you using a user 1000 everywhere without creating that user? It will be confusing when you login inside the container.

Reply:
I think I left this out in my last commit.

The Cassandra container is already using the `cassandra` user everywhere, with the UID 1000

That's why I was 1000:1000 everywhere. I'll change it to be "cassandra:cassandra" everywhere but that will still be confusing when compared with the `runAsUser: 1000` setting outside.